tests/test_SklearnGLMClassifierConverter.py

```diff
@@ -44,28 +49,36 @@ def test_model_logistic_regression_binary_class_nointercept(self):
                             allow_failure="StrictVersion(onnx.__version__) < StrictVersion('1.3') or "
                                           "StrictVersion(onnxruntime.__version__) <= StrictVersion('0.2.1')")
 
-    def ztest_model_logistic_regression_multi_class(self):
+    @unittest.skipIf(not onnx_built_with_ml(),
```

Comment:
when will the test suite run with a non-ml onnx package?

Reply:
ONNX 1.4.1 was released on pypi without this flag. SequenceType and DictionaryType are not available and it puzzled some people trying ONNX for the first time. The error message was not explicit.

Reply:
That's too bad, can we just add a checker in the converter main function to let people know this silly thing?

Reply:
Many converters work without onnx-ml. All classifiers fail but I prefer to have an explicit error message, or add a paragraph in the documentation instead of verifying this every time. 1.5.0 will be released with that flag and also for Python 3.7. The test will always pass.